src/views/auth/Signin.vue

```diff
@@ -1,22 +1,32 @@
 <template>
-  <sbc-signin @sync-user-profile-ready="onSessionReady()" />
+  <sbc-signin @sync-user-profile-ready="onProfileReady()" />
 </template>
 
 <script lang="ts">
 // Libraries
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Emit, Vue } from 'vue-property-decorator'
 
 // Components
 import SbcSignin from 'sbc-common-components/src/components/SbcSignin.vue'
 
+/**
+ * Operation:
+ * 1. When this component is first loaded (ie, we are not authenticated) then the
+ *    SbcSgnin component will redirect us to log in.
+ * 2. When this component is reloaded (ie, we are now authenticated) then the
+ *    SbcSignin component will emit "sync-user-profile-ready".
+ */
```

Comment:
Great comments here, I think this is very helpful for anyone not having worked with `Signin`.